tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -742,6 +774,15 @@ public:
          *fResultMin = std::min(m, *fResultMin);
    }
 
+   // Helper functions for RMergeableValue
+   void *GetMergeableValue()
+   {
+      const auto minfun = [](const ResultType &a, const ResultType &b) { return (b < a) ? b : a; };
```

Comment:
simply using `std::min` would be more readable

Reply:
As discussed, I will replace the body of the lambdas with `std::min` or `max`